net-misc/kea/files/kea-3.0.0-strip-doc.patch

```diff
@@ -0,0 +1,54 @@
+--- a/doc/sphinx/install-docs.sh.in
```

Comment:
Can you include some explanation for why we're doing this at the top of the patch?